test/util/common.go

```diff
@@ -485,3 +489,15 @@ func NewTestMsgUnDelegateValidator(address sdk.ValAddress, amt cosmosmath.Int) *
 	msg := stakingtypes.NewMsgUndelegate(sdk.AccAddress(address), address, sdk.NewCoin("stake", amt))
 	return msg
 }
+
+// ExecuteQGBHeights executes the end exclusive range of heights specified by beginHeight and endHeight
+// along with the QGB abci.EndBlocker on each one of them.
+// Returns the updated context with the correct height. Make sure to use it as the context in argument
+// will not have the last height as the one returned.
```

Comment:
Sorry I don't understand the last line here.

> Make sure to use it as the context in argument
> // will not have the last height as the one returned.

Reply:
removed, it was ambigeous